submodule-flakes-generated/generate_all_submodule_flakes.sh

```
@@ -0,0 +1,106 @@
+
+  outputs = { self, nixpkgs }:
+    let
+      system = "x86_64-linux";
+      pkgs = import nixpkgs {
+        inherit system;
+      };
+      submoduleUrl = "${submodule_url}";
+      submoduleRev = "${SUBMODULE_REV}";
+      submoduleSha256 = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+    in {
+      packages.${system}.default = pkgs.stdenv.mkDerivation {
```

Shellcheck (0.10.0)

[warning] system is referenced but not assigned.

(SC2154)